### PR TITLE
feat: backend-interaction

### LINKED ISSUE #3: Make Frontend talk to Backend
The User should be able to see test data processed by the backend.

### Acceptance Criteria

- frontend shows test data made available by the backend

### Story Points

- 5 Points

### Proposed completion in Iteration

- 2

### Marked as completed in Iteration

- 

### Actual Time in hours

### PR DESCRIPTION
**Pull request by Kvnstrck**
closes #3

### CODE REVIEW

**Review by nicosvv — APPROVED** [review]

**Review by D3vZro — CHANGES_REQUESTED** [review]
PR title does not conform to a conventional commit title.

File: visual-debugger/src/lib/components/ui/Button.svelte [hunk 1]
```diff
@@ -1,14 +1,17 @@
 <script lang="ts">
-    
+    // import function to call the bridge integration of queries to QueryBatchOverview
+    import {get_query_attributes} from "../../../data-processing/query-build";
+
     let { title } = $props();
 
-    function handleClick() {
-        console.log('Waiting for Instructions');
+    //call bridge to import the queries to QueryBatchOverview
```
Comment (Kha1r0) [inline visual-debugger/src/lib/components/ui/Button.svelte]: add space after //

**Review by MoPhliFra — APPROVED** [review]
**Looks good to me**

**Comment by Kvnstrck** [comment]
> PR title does not conform to a conventional commit title.

fixed

**Review by D3vZro — APPROVED** [review]

File: visual-debugger/src/lib/components/ui/Button.svelte (hunk 1 above)
Comment (Kvnstrck) [reply visual-debugger/src/lib/components/ui/Button.svelte]: fixed

**Review by Kha1r0 — APPROVED** [review]

**Pull request merged**